### PR TITLE
BSDT UX tweaks: adjust field widths and use UOM for Quantity label

### PR DESCRIPTION
**Pull request by bethbrains**
# Critical Changes

# Changes
- Bulk Service Delivery Tool field widths are adjusted to display more details on Client Name and Program Engagement fields
- Quantity field takes on the Unit of Measurement as its label upon save for clarity
image

# Issues Closed
- Fixes Issue with Long Labels overlapping text with the next field label.

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- [x] Code does not reference translatable strings in its logic
~~- [ ] Place holder data is incorporated (Refer to [Case Management Sample Data document](https://quip.com/cbFcAPJF8t0z))~~
~~- [ ] Base axe-core JEST test included for any new LWC (No critical violations)~~
~~- [ ] CRUD/FLS is enforced in Apex~~
~~- [ ] Permission sets are updated to account for CRUD/FLS for new object metadata~~
~~- [ ] All modified classes are unit tested (Apex) at 100% coverage~~
- [ ] UX approval or UX not necessary
- [x] PR contains draft release notes
- [x] Attach work item to the pull request with [Lurch](https://salesforce.quip.com/50ZRA5LEWVzH) `**Lurch:attach W-000000` or `**Lurch:add`
- [ ] All **acceptance criteria** have been met
    - [x] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by bethbrains** [comment]
**lurch: attach W-041040

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HvLWQA0" target="_blank">W-041040</a>

**Comment by jjbennett** [comment]
**lurch: attach W-040302

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HtrhQAC" target="_blank">W-040302</a>

**Comment by jjbennett** [comment]
@bethbrains - after playing around with increasing the label size, I realized that although our fields are now good, there is still potential for the underlying overlap issue. I just pushed a change for slds-truncate to this branch. Figured it'd save QE a bit of setup for the same focus area. Thanks!

File: force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.js [hunk 1]
```diff
@@ -261,6 +264,8 @@ export default class ServiceDeliveryRow extends LightningElement {
         this.handleSaveEnd();
         this.lockContactField();
         fireEvent(this.pageRef, "serviceDeliveryUpsert", event.detail);
+        this.unitOfMeasureValue =
+            event.detail.fields[this.fields.unitMeasurement.fieldApiName].value;
```
Comment (jjbennett) [inline force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.js:268]: What happens if they do not have permission to this field?

File: force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.js [hunk 2]
```diff
@@ -31,10 +31,12 @@ import selectEngagement from "@salesforce/label/c.Select_Program_Engagement";
 import selectedContactWarning from "@salesforce/label/c.Service_Delivery_Contact_Without_Programs";
 import noServiceWarning from "@salesforce/label/c.No_Services_For_Program_Engagement";
 import newProgramEngagement from "@salesforce/label/c.New_Program_Engagement";
+import quantity from "@salesforce/label/c.Quantity";
 
 import CONTACT_FIELD from "@salesforce/schema/ServiceDelivery__c.Contact__c";
 import SERVICE_FIELD from "@salesforce/schema/ServiceDelivery__c.Service__c";
 import PROGRAMENGAGEMENT_FIELD from "@salesforce/schema/ServiceDelivery__c.ProgramEngagement__c";
+import UNITMEASUREMENT_FIELD from "@salesforce/schema/ServiceDelivery__c.UnitOfMeasurement__c";
```
Comment (jjbennett) [inline force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.js:39]: Should we have an underscore between UNIT and MEAS..?

**Pull request merged**